Cargo.toml

```diff
@@ -4,6 +4,7 @@ members = [
     "nomos-da/kzgrs",
     "nomos-da/kzgrs-backend",
     "nomos-da/storage",
+    "nomos-da/subnets",
```

Comment:
I would prefer to call it `network`

Reply:
Maybe `dispersal`? As it would encapsulate the executors dispersing and da nodes replicating inside the subnets?

Reply:
I think this will be a core of the da-network. We will have stuff for dispersal and sampling inside. Also it was  a good choice to have this separated crate as it can be part of the sdk later on. So, i think we will have:
```
nomos-da /network
    /core
        /backends
            /libp2p
    /assignation
    /sampling
    /dispersal
```
More or less. Does ti makes sense?

Reply:
This makes sense, I'll move this code in this pr to `nomos-da/network/dispersal` crate `nomos-da-dispersal`.